src/components/common/ColonyHome/ColonyMembersWidget/MembersSubsection.tsx

```diff
@@ -150,39 +144,36 @@ const MembersSubsection = ({
     <div className={styles.main}>
       {setHeading(true)}
       <ul className={styles.userAvatars}>
-        {(colonyWatchers as { user: User }[])
-          .slice(0, avatarsDisplaySplitRules)
-          .map(({ user }) => (
-            <li className={styles.userAvatar} key={user.walletAddress}>
-              <UserAvatar
-                size="xs"
-                address={user.walletAddress}
-                // banned={canAdministerComments && banned}
-                banned={false}
-                showInfo
-                colony={colony}
-                user={user}
-                popperOptions={{
-                  placement: 'bottom',
-                  showArrow: false,
-                  modifiers: [
-                    {
-                      name: 'offset',
-                      options: {
-                        /*
-                         * @NOTE Values are set manual, exactly as the ones provided in the figma spec.
-                         *
-                         * There's no logic to how they are calculated, so next time you need
-                         * to change them you'll either have to go by exact specs, or change
-                         * them until it "feels right" :)
-                         */
-                        offset: [-208, -12],
-                      },
+        {members.slice(0, avatarsDisplaySplitRules).map((member) => (
+          <li className={styles.userAvatar} key={member?.user?.walletAddress}>
+            <UserAvatar
+              size="xs"
+              address={member?.user?.walletAddress || ''}
+              // banned={canAdministerComments && banned}
+              showInfo
+              colony={colony}
+              user={member?.user}
```

Comment:
It doesn't look like `member` can currently be undefined so could do without the optional chaining 
If you do go with my suggestion above, then you can run `filter(notNull)` on the `members` array to remove null items

Reply:
Nice, didn't know about notNull ~utils/arrays